src/main/java/uk/gov/justice/hmpps/prison/api/model/CourtEventDto.kt

```diff
@@ -0,0 +1,48 @@
+package uk.gov.justice.hmpps.prison.api.model
+
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+data class CourtEventDto(
+  var offenderNo: String? = null,
```

Comment:
would have liked to use the DataClassRowMapper to call the constructor here, but found that calling `getColumn(name)` in Oracle was ridiculously slow so binned the idea (would have required to rewrite the mapper).  Therefore have to make all fields nullable and also `var` so that each property can then be set.